I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 187, in script
    $ raccoonanger += 1
  File "game/script.rpy", line 187, in <module>
    $ raccoonanger += 1
NameError: name 'raccoonanger' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 187, in script
    $ raccoonanger += 1
  File "/Users/Kossio/Downloads/renpy-6.18.3-sdk/renpy/ast.py", line 785, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/Kossio/Downloads/renpy-6.18.3-sdk/renpy/python.py", line 1382, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 187, in <module>
    $ raccoonanger += 1
NameError: name 'raccoonanger' is not defined

Darwin-14.0.0-x86_64-i386-64bit
Ren'Py 6.18.3.761
garbagepals 0.0
